index.html

```diff
@@ -334,6 +334,14 @@ <h3>
     is set to an [=implementation-defined=] value, greater than 0.
     </li>
   </p>
+  <p>
+    A constructed  {{PressureRecord}} object has the following internal slots:
```

Comment:
nit: you've got an extra space character before `{{PressureRecord}}`.

Reply:
fixed